machinelearning/tensorflow.md

```diff
@@ -2,60 +2,51 @@
 
 **Introduction**
 
-TensorFlow is an open-source software library for machine learning and artificial intelligence. It can be used across training and inference of deep neural networks. This document covers how to use TensorFlow based machine learning inference on Graviton CPUs, what runtime configurations are important and how to debug any performance issues. The document also covers instructions for source builds and how to enable some of the downstream features.
+TensorFlow is an open-source software library for machine learning and artificial intelligence. It can be used across training and inference of deep neural networks. This document covers how to use TensorFlow based machine learning inference on Graviton CPUs, what runtime configurations are important and how to debug any performance issues.
 
 # How to use TensorFlow on Graviton CPUs
-There are multiple levels of software package abstractions available: Python wheel (easiest option), Docker container (comes with the wheel, additional packages and benchmarks), DLAMI (Deep Learning Amazon Machine Image that comes with preinstalled packages and tools) and AWS DLC (Deep Learning Contianers with TensorFlow Serving API interface). Examples of using each method are below.
+There are multiple levels of software package abstractions available: AWS DLC (Deep Learning Container, gives the best performance, comes with additional optimizations on top of the official wheels, and all the packages installed), Python wheel (easiest option to get the release features), and the Docker hub images (comes with downstream experimental features). Examples of using each method are below.
```

Comment:
Bulleted list or tables might be easier to parse than in-line with parentheses.

Reply:
thanks, updated it.